mailing_list/templates/mailing_list/admin_index.html

```diff
@@ -62,110 +66,109 @@
                   </a>
               </div>
           </p>
+          <p class="caption">
+            If you would like a copy of your email, add your own email address to the CC or BCC field.
+          </p>
         </div>
 
+    <div class="col-md-12">
+
+      <!-- show combined mailing list or single primary list -->
+      <!-- if there is a combined mailing list show it -->
+      <h2 ng-show="ml.hasCourseEmailList() || ml.enrollmentSectionLists.length == 1">Email Entire Course</h2>
+
+      <div ng-repeat="list in ml.courseList">
+        {% include "mailing_list/_mailing_list_details.html" with scope="class" %}
         <div class="col-md-12">
-            <!--<hr width="98%">-->
-            <!-- show combined mailing list or single primary list -->
-            <!-- if there is a combined mailing list show it -->
-            <div ng-if="ml.isCourseList(list)" ng-repeat="list in ml.primarySectionLists">
-              {% include "mailing_list/_mailing_list_details.html" with scope="class" %}
-                <div class="col-md-12">
-                    <hr width="98%">
-                </div>
-            </div>
+          <hr width="98%">
+        </div>
+      </div>
 
-            <!-- otherwise show the primary section -->
-            <div ng-if="ml.primarySectionLists.length == 1" ng-repeat="list in ml.primarySectionLists">
-              {% include "mailing_list/_mailing_list_details.html" with scope="class" %}
+      <div ng-if="ml.enrollmentSectionLists.length == 1" ng-repeat="list in ml.enrollmentSectionLists">
+        {% include "mailing_list/_mailing_list_details.html" with scope="class" %}
+        <div class="col-md-12">
+          <hr width="98%">
+        </div>
+      </div>
+
+      <div class="btn-group col-md-12">
+        <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
+          <div class="panel panel-default" ng-show="ml.hasNonEnrollmentSections()">
+            <div class="panel-heading" role="tab" id="headingOne">
+              <h4 class="panel-title">
+                <a class="collapsed"
+                 role="button"
+                 data-toggle="collapse"
+                 data-parent="#accordion"
+                 href="#collapseTwo"
+                 aria-expanded="false" aria-controls="collapseTwo">
+                  <ng-pluralize count="ml.nonEnrollmentSectionLists.length"
+                        when="{'0': 'No Sections', 'one': 'Email Specific Sections', 'other': 'Email Specific Sections'}">
```

Comment:
Is the phrase  'Email Specific Sections' intended for both conditions- when length is 1 or greater?  In which case, maybe the check for one is redundant?